azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/view_model/ChatMessageViewModel.kt

```diff
@@ -0,0 +1,5 @@
+package com.azure.android.communication.ui.chat.presentation.ui.view_model
+
+import com.azure.android.communication.chat.models.ChatMessage
```

Comment:
We can use `MessageInfoModel`

Reply:
I'll take a look, ViewModel may be different

Reply:
Will update.

Reply:
Going to use MessageInfoModel in MessageViewModel, and List<MessageInfoModel> as type for MessageRepository